packages/calcite-components/src/components/list-item/list-item.e2e.ts

```diff
@@ -370,7 +382,7 @@ describe("calcite-list-item", () => {
     expect(calciteListItemClose).toHaveReceivedEventTimes(1);
   });
 
-  it("should fire calciteListItemToggle event when opened and closed", async () => {
+  it("should fire calciteListItemToggle event when expanded and collapsed", async () => {
```

Comment:
Can you keep the open/closed tests grouped in a "deprecated" describe block? Bonus points if you can extract the tests to be reused between both.

Reply:
Updating for record here: As we decided, because this ends up touching on too many tests, we'll instead just add a test to check if setting `open` sets `expanded`.